src/remotebuild/package.json

```diff
@@ -34,6 +34,7 @@
         "q": "1.0.1",
         "rimraf": "2.2.6",
         "semver": "^4.3.6",
+		"colors": "^1.0.3",
```

Comment:
Looks like this is only used for a test, so I think it should be a devdependency